include/bluetooth/bluetooth.h

```diff
@@ -486,13 +486,28 @@ void bt_data_parse(struct net_buf_simple *ad,
 		   bool (*func)(struct bt_data *data, void *user_data),
 		   void *user_data);
 
+/** OOB data that is specific for LE SC pairing method. */
+struct bt_le_sc_oob_data {
+	/** Random Number. */
+	u8_t r[16];
+
+	/** Confirm Value. */
+	u8_t c[16];
+};
+
+/** General OOB data. */
 struct bt_le_oob {
 	/** LE address. If local privacy is enabled this is Resolvable Private
 	 *  Address.
 	 */
 	bt_addr_le_t addr;
+
+	/** OOB data that are relevant for LESC pairing. */
+	struct bt_le_sc_oob_data le_sc_data;
 };
 
+struct bt_conn;
```

Comment:
This looks a bit strange in `bluetooth.h`. Until now the intention has been to keep it independent of any connection stuff.

Reply:
fixed when moving relevant APIs to conn, this dependency is no longer required